lib/logger.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import logging.config
+
+class SeleniumFilter(logging.Filter):
+    def filter(self, record):
+        selenium_log = 'selenium.webdriver.remote.remote_connection'
+        return selenium_log not in record.getMessage()
+
+# much of this taken from http://victorlin.me/posts/2012/08/26/good-logging-practice-in-python
+# reference -- https://docs.python.org/2/library/logging.config.html
+logging.config.dictConfig({
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "simple": {
+            "format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        },
+        "console": {
+            "format": "%(message)s"
+        }
```

Comment:
These are the formatters. The `"simple"` formatter is used in the `*.log` files that are written to disk. They have a lot more information in them than the `"console"` formatter. It just prints out the string passed into the logger call, almost how `print` works.